.gitignore

```diff
@@ -73,3 +73,5 @@ Thumbs.db
 npm-debug.log*
 yarn-debug.log*
 yarn-error.log*
+
+yarn.lock
```

Comment:
this shouldn't be ignored because if the repo has continue delivering its necessary to run the scripts